docker/Dockerfile

```diff
@@ -0,0 +1,68 @@
+############################################
+# Dockerfile to run the PCIC data portal   #
+# Based on Ubuntu 16.04                    #
+############################################
+
+FROM ubuntu:16.04
+MAINTAINER Carl Masri <cmasri@uvic.ca>
+
+# Get the system level dependencies
+RUN apt-get update && apt-get install -y \
+    python-dev \
+    python-virtualenv \
+    build-essential \
+    libhdf5-dev \
+    libgdal-dev \
+    libnetcdf-dev \
+    git
+
+# Set up environment variables
+ENV CPLUS_INCLUDE_PATH /usr/include/gdal
+ENV C_INCLUDE_PATH /usr/include/gdal
+ENV PDP_CONFIG /root/pdp_config.yaml
+ENV PATH /root/pdp/pyenv/bin:$PATH
+ARG BRANCH=dev
+
+# Clone repo and set up virtualenv
+WORKDIR /root/
+RUN git clone https://github.com/pacificclimate/pdp -b ${BRANCH}
```

Comment:
So, I don't think that this is what we discussed over e-mail. We want docker to build whatever branch is currently checked out. The easiest way to do this is to just ADD the source dir to the image, like we do [here](https://github.com/pacificclimate/climate-explorer-backend/blob/master/Dockerfile#L3).

This will also mean that you don't need to install git in the system depends.